BlockParty.py

```diff
@@ -262,7 +263,7 @@ def key_action(self, key):
                 else:  # if key in enter_key:
                     self.queue_draw_complete()
                     self.next_tick = time.time() + self.time_step
-                    self.game_mode = self.PLAY
+                    self.game_mode = self.PLAY  
```

Comment:
Extra spaces at the end, which would fail flake8 tests.

Please look at the diffs of your change before making a commit.